### ISSUE TITLE
Misclassification child chest X-Ray

### ISSUE THREAD

**Issue by auriml** [xinario/chestViewSplit#1]
Hi, 
I have forked your project to adapt it to python 3.6.1 and pytorch 0.3.1 and try classifying images in other formats (different sizes and uint16). I have uploaded a sample of the new images in [https://github.com/auriml/chestViewSplit](https://github.com/auriml/chestViewSplit/tree/master/chest_xray/chest_xray_new) and implemented a custom loader. It works as expected as far as I have tested it except for this child chest X-ray [chestViewSplit/chest_xray/chest_xray_new/7wa3ao.png](https://github.com/auriml/chestViewSplit/blob/master/chest_xray/chest_xray_new/7wa3ao.png) . Any hint?  Thanks!!

**Comment by xinario** [comment]
My gut feeling is the large padding around the image due to the small size of the child. In my training data got from openi, many side view X-rays have larger padding than front view ones. 

So Just try to remove the padding and test it again to see how it performs.
